serverUtils/serverUtil.tsx

```diff
@@ -37,9 +36,3 @@ export function hexToBytes(hex: string): Uint8Array {
 export function bytesToHex(bytes: Uint8Array): HexString {
     return Buffer.from(bytes).toString("hex") as HexString
 }
-
-export function getMd5HashBase64(data: string): Base64String {
-    return createHash("md5")
-        .update(data, "utf-8")
-        .digest("base64") as Base64String
-}
```

Comment:
Not saying we have to do this, but it would actually be nice to keep `md5-to-hex` around here and use it instead of the `md5` module we're currently using in a few places.